instrumentation/geode-1.4/src/main/java/io/opentelemetry/javaagent/instrumentation/geode/GeodeQueryNormalizer.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.javaagent.instrumentation.geode;
+
+import io.opentelemetry.javaagent.instrumentation.api.db.normalizer.ParseException;
+import io.opentelemetry.javaagent.instrumentation.api.db.normalizer.SqlNormalizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public final class GeodeQueryNormalizer {
+  private static final Logger log = LoggerFactory.getLogger(GeodeQueryNormalizer.class);
+
+  public static String normalize(String query) {
```

Comment:
If this is just a simple wrapper, why do we need it?

Reply:
Just for the try-catch right now. In the (close) future we'll also check if the normalization is enabled (I'll add appropriate config flags in https://github.com/open-telemetry/opentelemetry-java-instrumentation/issues/1431)